src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/AttachedCollections/Implementation/FrameworkReferenceAssemblyItem.cs

```diff
@@ -38,7 +38,11 @@ public FrameworkReferenceAssemblyItem(string assemblyName, string? path, string?
 
         public override object? GetBrowseObject() => new BrowseObject(this);
 
+        string? IObjectBrowserItem.AssemblyPath => GetAssemblyPath();
 
+        private string? GetAssemblyPath() => Path is not null
+            ? System.IO.Path.GetFullPath(System.IO.Path.Combine(Framework.Path, Path))
```

Comment:
just curious, how does this path look like? are any of the paths rooted?

Reply:
It ultimately ends up as an absolute path to the assembly, so that we can use it in the object browser and show it in the "Properties" view. The `Framework.Path` is something like `C:\Program Files\dotnet\packs\Microsoft.NETCore.App.Ref\6.0.36` and the `Path` is something like `ref\net6.0\Microsoft.CSharp.dll`. The `GetFullPath` just makes the path look normal (i.e. consistent `/` vs `\` characters).